src/hotspot/share/gc/g1/g1FullGCMarker.hpp

```diff
@@ -63,6 +63,8 @@ class G1FullGCMarker : public CHeapObj<mtGC> {
   G1FollowStackClosure _stack_closure;
   CLDToOopClosure      _cld_closure;
 
+  G1RegionMarkStatsCache _mark_region_cache;
```

Comment:
Could you call this instance `_mark_stats_cache` too like in the other use, and adapt the various method names? `_mark_region_cache` does not indicate that this has something to do with (liveness) "statistics" cache. I do not know what a "region cache" is.

Reply:
pr #2966